man/systemd.network.xml

```diff
@@ -688,6 +688,56 @@
             which is then configured to use them explicitly.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>Home=</varname></term>
+          <listitem>
+            <para>Designates this address the "home address" as defined in
+            <ulink url="https://tools.ietf.org/html/rfc6275">RFC 6275</ulink>.
+            Works only for IPv6.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>NoDAD=</varname></term>
+          <listitem>
+            <para>Do not perform Duplicate Address Detection
+            <ulink url="https://tools.ietf.org/html/rfc4862">RFC 4862</ulink> when adding this address.
+            Works only for IPv6.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>ManageTempAddr=</varname></term>
+          <listitem>
+            <para>ManageTempAddr make the kernel manage temporary addresses created
+            from this one as template on behalf of Privacy Extensions
+            <ulink url="https://tools.ietf.org/html/rfc3041">RFC 3041</ulink>.
+            For this to become active, the use_tempaddr sysctl setting has to be set
+            to a value greater than zero. The given address needs to have a prefix
+            length of 64. This flag allows to use privacy extensions in a manually
+            configured network, just like if stateless auto-configuration was active.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>NoPrefixRoute=</varname></term>
+          <listitem>
+            <para>When adding or modifying an IPv6 address, the userspace application needs
```

Comment:
NoPrefixRoute=yes → PrefixRoute=no.

"Takes a boolean argument…". "The default is …".

Please rewrite the paragraph to first state what this option does, and only then explain when it is useful.